### DISCUSSION TITLE
Release 5.2.0 : Private repo variables missing depoyment

### DISCUSSION THREAD

**Discussion by gohilamariappan** [Sunbird-Ed/Community#551] [Bugs]
Hi team!
Below are the variables which are not giving error in the 5.2.0 Core/KP deployments
@santhosh-tg  please check
##########3
sunbird_public_storage_account_name: ""
sunbird_public_storage_region: ""
sunbird_private_storage_account_key:""

sunbird_public_storage_account_name: ""
sunbird_public_storage_region: ""

cloud_storage_cname_url: ""

sunbird_private_storage_account_name: ""
sunbird_private_storage_region: ""
sunbird_private_storage_account_name: ""
sunbird_private_storage_region: ""
sunbird_content_azure_storage_container: " "
sunbird_public_storage_account_key: ""

Regards
Gohila

**Comment by santhosh-tg** [comment]
@gohilamariappan  Can you share the jenkins jobs which was having these issues ?

**Comment by gohilamariappan** [comment]
@santhosh-tg Below one is the job.
Deploy/dev/Lern/LernDataProducts

**Reply by santhosh-tg** [reply]
Hi @AmiableAnil @kumarks1122 can you please check on this?

**Reply by AmiableAnil** [reply]
@gohilamariappan which version on the lern-data-products are you using?

**Comment by gohilamariappan** [comment]
@AmiableAnil we are configuring release-5.2.0 version. In that Lern services branch release-5.1.0_RC1.

Regards,
Gohila

**Reply by AmiableAnil** [reply]
@gohilamariappan Please add the following variable in private repo common.yml

sunbird_private_storage_account_name: "{{ cloud_private_storage_accountname }}"
sunbird_private_storage_account_key: "{{ cloud_private_storage_secret }}"
sunbird_public_storage_account_name: "{{ cloud_public_storage_accountname }}"
sunbird_public_storage_account_key: "{{ cloud_public_storage_secret }}"

**Reply by gandham-santhosh** [reply]
@gohilamariappan can you please mark this thread if it is resolved?

**Comment by gohilamariappan** [comment]
@AmiableAnil & @gandham-santhosh   Thank you so much!  Issue resolved.
